microprofile/WFLY-19592_MicroProfile_Fault_Tolerance_4_1.adoc

```diff
@@ -0,0 +1,127 @@
+To certify functionality, the MP FT TCK is run as part of the WildFly testsuite.
+
+=== User Stories
+
+User wants to deploy an application using MicroProfile Fault Tolerance 4.1 from MicroProfile Platform 7.0.
+
+== Issue Metadata
+
+* https://issues.redhat.com/browse/WFLY-19592[WFLY-19592 [MP Platform 7] MicroProfile Fault Tolerance 4.1]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-19592[WFLY-19588 Implement MicroProfile Platform 7] – MP Platform parent issue
+* https://issues.redhat.com/browse/WFLY-19590[WFLY-19590 MicroProfile Telemetry 2.0] – dependency of this issue for full Telemetry metrics integration
+
+// EAP Tracking issue:
+// * https://issues.redhat.com/browse/EAP7-2225[EAP7-2225 [MP Platform 7] MicroProfile Fault Tolerance 4.1]
+
+=== Affected Projects or Components
+
+* Eclipse MicroProfile Fault Tolerance
+* SmallRye Fault Tolerance
+
+=== Other Interested Projects
+
+* N/A
+
+=== Relevant Installation Types
+
+* Traditional standalone server (unzipped or provisioned by Galleon)
+* OpenShift Source-to-Image (S2I)
+* Bootable jar
+
+== Requirements
+
+* Upgrade MP FT specification modules to 4.1.
+* Upgrade SmallRye FT to 6.5.x.
+* Enable integration with MP Telemetry Metrics when the subsystem is available in the serve configuration.
+
+=== Changed requirements
+
+* N/A
+
+=== Non-Requirements
+
+* N/A
+
+=== Future Work
+
+None.
+
+== Backwards Compatibility
+
+The specification and the implementation do not introduce any incompatibilities from the previous version (MP FT 4.0).
+
+=== Default Configuration
+
+There are no changes to the existing configuration.
+
+=== Importing Existing Configuration
+
+The subsystem does not have any configuration, so existing subsystem configuration is not effected.
+
+=== Deployments
+
+The specification and the implementation do not introduce any incompatibilities from the previous version relevant to deployments.
+
+=== Interoperability
+
+* N/A
+
+== Implementation Plan
+
+For the integration with Telemetry metrics to function,
+the integration of its https://issues.redhat.com/browse/WFLY-19590[subsystem and modules] is required.
+Implementation will be done in the created `mp-7-wip` https://github.com/wildfly/wildfly/tree/mp-7-wip[branch] in WildFly upstream repository.
+
+== Admin Clients
+
+* No effect on the existing clients, there are no model changes.
+
+== Security Considerations
+
+* No security implications of this integration.
+
+[[test_plan]]
+== Test Plan
+
+MicroProfile Fault Tolerance TCK tests will be added and executed by CI as part of the WildFly testsuite (in `testsuite/integration/microprofile-tck/fault-tolerance` module)
+WildFly-specific behavior and configuration is covered by existing tests in the WildFly testsuite (in `testsuite/integration/microprofile` module)
+
+== Community Documentation
+
+References to the MP FT specifications in the existing documentation will be updated to reflect the updated versions.
```

Comment:
Should we log a WFLY to target updates to the Quickstarts?

Reply:
There are no plans to update the QS at this time and no changes are required either. 

When the quickstarts projects pulls in new WildFly version (i.e. 35) it will be running with the update version with no changes to the QS itself.